.changeset/chilly-shrimps-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-foundations': patch
+---
+
+- Introduced a CSS variable: `--source-text-decoration-thickness`, provided by the typography API
```

Comment:
How do you feel about reordering this sentence?
```suggestion
- Typography functions now include a `--source-text-decoration-thickness` CSS variable
```
This refines the scope as you go, rather than starting with the micro level (the change) and building up to the macro level (the area that includes the change): 

area (Typography functions) > type of change (now include) > the change (a `--source-text-decoration-thickness` CSS variable)